doc/community-build/docker/cortx-all/README.md

```diff
@@ -82,15 +82,23 @@ This document provides step-by-step instructions to build required binaries and
     docker run --name release-packages-server -v /var/artifacts/0/:/usr/share/nginx/html:ro -d -p 80:80 nginx
     ```
 
-7. We need to change docker compose file and add below extra_hosts for all the services like below. 
+7. Once docker container is up and running, We need to clone cortx repo use below commands for same.
+
+    ```
+    docker ps 
+    git clone https://github.com/Seagate/cortx-re
```

Comment:
To make it simple
`git clone https://github.com/Seagate/cortx-re && cd cortx-re/docker/cortx-deploy/ `